src/index.js

```diff
@@ -12,16 +13,18 @@ server.express.use(compression());
 
 const config = require('../config');
 
+server.express.use(cors({
+  origin: true,
```

Comment:
Graphql-yoga already relies on the cors under the hood. Instead of introducing it again, can we just add this to line 27?. 

Alternatively, we can introduce an array of whitelisted domains https://github.com/prisma-labs/graphql-yoga/blob/master/src/index.ts#L212

Reply:
Oh yes. I understand what you mean. However, I think it would still work with this configuration. This is the specific issue addressed [on this issue](https://github.com/expressjs/cors/issues/134#issuecomment-413543241).
Graphql-yoga won't have a problem with this config.